spec/helper-spec.js

```diff
@@ -48,7 +48,7 @@ describe('linter helpers', function () {
         expect(range[0][0]).toEqual(7)
```

Comment:
Why not condense these down to a `toEqual(` like the others? (Not necessary, but might as well be consistent)